homeassistant/components/fibaro/__init__.py

```diff
@@ -406,6 +406,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
         identifiers={(DOMAIN, controller.hub_serial)},
+        serial_number=controller.hub_serial,
```

Comment:
It's a bit strange that now the same value is added for model and serial_number. Probably we can merge this change like this and I can later try to find a better information which can be added as model. For newer models there is such an information (HC3, HC3L, YH), but for older no model information is available.

Reply:
> For newer models there is such an information (HC3, HC3L, YH), but for older no model information is available.

For older models we should, in that case, not provide model information :)